flocker/node/functional/test_deploy.py

```diff
@@ -35,8 +35,10 @@ def test_restart(self):
         """
         name = random_name()
         docker_client = DockerClient()
-        deployer = Deployer(create_volume_service(self), docker_client,
-                            make_memory_network())
+        deployer = P2PNodeDeployer(
+            u"localhost",
```

Comment:
:question: Our docs say "hostname" all over the place but in practice the value is always going to be an IP address.